docs/pages/eas/workflows/pre-packaged-jobs.mdx

```diff
@@ -0,0 +1,812 @@
+| app_build_version | string | The version code/build number of the app.                          |
+| app_identifier    | string | The bundle identifier/package name of the app.                     |
+| app_version       | string | The version of the app.                                            |
+| channel           | string | The update channel used for the build.                             |
+| distribution      | string | The distribution method used. Can be `internal` or `store`.        |
+| fingerprint_hash  | string | The fingerprint hash of the build.                                 |
+| git_commit_hash   | string | The git commit hash used for the build.                            |
+| platform          | string | The platform the build was created for. Either `android` or `ios`. |
+| profile           | string | The build profile used.                                            |
+| runtime_version   | string | The runtime version used.                                          |
+| sdk_version       | string | The SDK version used.                                              |
+| simulator         | string | Whether the build is for simulator.                                |
+
+### Examples
+
+Here are some practical examples of using the build job:
+
+<Collapsible summary="Basic build for a specific platform">
+
+This workflow builds your iOS app whenever you push to the main branch.
+
+```yaml .eas/workflows/build-ios.yml
+name: Build iOS app
+
+on:
+  push:
+    branches: ['main']
+
+jobs:
+  build_ios:
+    name: Build iOS
+    type: build
+    params:
+      platform: ios
+      profile: production
+```
+
+</Collapsible>
+
+<Collapsible summary="Build for both platforms in parallel">
+
+This workflow builds both Android and iOS apps in parallel when you push to the main branch.
+
+```yaml .eas/workflows/build-all.yml
+name: Build for all platforms
+
+on:
+  push:
+    branches: ['main']
+
+jobs:
+  build_android:
+    name: Build Android
+    type: build
+    params:
+      platform: android
+      profile: production
+
+  build_ios:
+    name: Build iOS
+    type: build
+    params:
+      platform: ios
+      profile: production
+```
+
+</Collapsible>
+
+<Collapsible summary="Build with environment variables">
+
+This workflow builds your Android app with custom environment variables that can be used during the build process.
+
+```yaml .eas/workflows/build-with-env.yml
+name: Build with environment variables
+
+on:
+  push:
+    branches: ['main']
+
+jobs:
+  build_android:
+    name: Build Android
+    type: build
+    env:
+      APP_ENV: production
+      API_URL: https://api.example.com
+    params:
+      platform: android
+      profile: production
+```
+
+</Collapsible>
+
+<Collapsible summary="Build with different profiles">
+
+This workflow creates two different Android builds using different profiles - one for internal distribution and one for store submission.
```

Comment:
Should we note that a prerequisite for this to work is existence of `internal` and `store` profiles in `eas.json`?